src/System.Collections/tests/Generic/List/List.Generic.Tests.Constructor.cs

```diff
@@ -3,20 +3,21 @@
 // See the LICENSE file in the project root for more information.
 
 using System.Collections.Generic;
-using System.Linq;
+using System.Collections.Tests;
+using System.Runtime.CompilerServices;
 using Xunit;
 
-namespace System.Collections.Tests
+namespace System.Linq.Expressions.Tests
 {
     /// <summary>
     /// Contains tests that ensure the correctness of the List class.
     /// </summary>
-    public abstract partial class List_Generic_Tests<T> : IList_Generic_Tests<T>
+    public abstract partial class ReadOnlyCollectionBuilder_Generic_Tests<T> : IList_Generic_Tests<T>
     {
         [Fact]
         public void Constructor_Default()
         {
-            List<T> list = new List<T>();
+            ReadOnlyCollectionBuilder<T> list = new ReadOnlyCollectionBuilder<T>();
```

Comment:
Why did you make these changes? This is removing test coverage for `List<T>`. Instead, you need to change the tests here: https://github.com/dotnet/corefx/blob/master/src/System.Linq.Expressions/tests/ReadOnlyCollectionBuilderTests.cs